pkg/manifestcomparators/comp_must_not_exceed_cost_budget.go

```diff
@@ -109,15 +142,20 @@ func (b mustNotExceedCostBudget) Validate(crd *apiextensionsv1.CustomResourceDef
 
 				return false
 			})
+
+		if rootCELContext != nil && rootCELContext.TotalCost != nil && rootCELContext.TotalCost.Total > apiextensionsvalidation.StaticEstimatedCRDCostLimit {
```

Comment:
Can we add an extra test (with multiple expensive fields) specifically for the total cost?